libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -353,6 +358,7 @@ public void startNavigation(NavigationViewOptions options) {
     clearMarkers();
     // Initialize navigation with options from NavigationViewOptions
     if (!isInitialized) {
+      initLocale(options);
       navigationViewModel.initializeNavigationOptions(getContext().getApplicationContext(),
         options.navigationOptions().toBuilder().isFromNavigationUi(true).build());
       // Initialize the camera (listens to MapboxNavigation)
```

Comment:
At this point, we can pass the `locale` / `unitType` to `InstructionView` / `SummaryBottomSheet` as those views have already been inflated